components/Alert/Alert.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { cva, type VariantProps } from 'class-variance-authority';
+
+import { cn } from '@/utils/utils';
+
+const alertVariants = cva(
+  'relative w-full rounded-lg border p-4 [&>svg~*]:pl-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
+  {
+    variants: {
+      variant: {
+        default: 'bg-background text-foreground',
+        error:
+          'bg-background text-foreground [&>svg]:text-red-500 text-red-500 border-red-500',
+        warning:
+          'bg-background text-foreground [&>svg]:text-yellow-500 text-yellow-500 border-yellow-500',
+        success:
+          'bg-background text-foreground [&>svg]:text-green-500 text-green-500 border-green-500',
+      },
+    },
+    defaultVariants: {
+      variant: 'default',
+    },
+  },
+);
+
+const Alert = React.forwardRef<
+  HTMLDivElement,
+  React.HTMLAttributes<HTMLDivElement> & VariantProps<typeof alertVariants>
+>(({ className, variant, ...props }, ref) => (
+  <div
+    ref={ref}
+    role="alert"
+    className={cn(alertVariants({ variant }), className)}
+    {...props}
+  />
+));
+Alert.displayName = 'Alert';
+
+const AlertTitle = React.forwardRef<
```

Comment:
Move AlertTitle and AlertDescription into their own files. I know Shadcn created these in here, but they should be separated into files.